adapters/syslog/syslog.go

```diff
@@ -81,13 +81,13 @@ type SyslogAdapter struct {
 func (a *SyslogAdapter) Stream(logstream chan *router.Message) {
 	defer a.route.Close()
 	for message := range logstream {
-		m := &SyslogMessage{message, a.conn}
+		m := NewSyslogMessage(message, a.conn)
 		buf, err := m.Render(a.tmpl)
 		if err != nil {
 			log.Println("syslog:", err)
 			return
 		}
-		_, err = a.conn.Write(buf)
+		_, err = a.conn.Write(buf.Bytes())
```

Comment:
It's not a fix, it's part of a change required by this PR that helps generalize syslog.